### DISCUSSION TITLE
PC98用ゲーム「コープスパーティー」をLinuxで遊ぶ - わたすけのへや

### DISCUSSION THREAD

**Discussion by giscus** [watasuke102/watasuke.net#2] [Article]
# PC98用ゲーム「コープスパーティー」をLinuxで遊ぶ - わたすけのへや

PC98エミュレータのセットアップ方法を解説します（Windowsにも対応しています）

https://watasuke.net/blog/article/pc98-emu-on-linux/

**Comment by zibuhu** [comment]
06.起動するのコマンドを入力しても構文エラーですと出てきます。　　　　　　　　　　　　　　　　　　　　　　　　　　　　　　　　　　　　　　　　　　　　どうすればよいですか？

**Reply by watasuke102** [reply]
`dir` コマンドなどを使って、フォルダ名や.batファイルの存在を確認してみてください。

実際にどのような操作をして、どのようなエラーが出たのか、というのがわかるスクリーンショット等があれば見せてください！

**Comment by HUHUGOKI** [comment]
コマンドは改行しますか？

**Reply by watasuke102** [reply]
1行あたり1コマンドとなっています（Enterを押すと実行されるので、改行という概念はないです）
